seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/base/source/IncrementalSource.java

```diff
@@ -76,6 +79,14 @@
     protected StopMode stopMode;
     protected DebeziumDeserializationSchema<T> deserializationSchema;
 
+    protected SeaTunnelDataType<SeaTunnelRow> dataType;
+    protected ReadonlyConfig options;
+
+    protected IncrementalSource(ReadonlyConfig options, SeaTunnelDataType<SeaTunnelRow> dataType) {
+        this.options = options;
+        this.dataType = dataType;
```

Comment:
The `#prepare` method will not be called in the new process, please initialize the remaining `fields`.